### PR TITLE
14026 Changes for OneStop

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#14026

*Description of changes:*

- changed OneStop to grayed out with a Moved badge
- updated copy for OneStop and Business Registry
- deleted references to OneStop URL
- fixed some lint warnings
- app version = 1.0.24

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Review by pwei1018 — APPROVED** [review]

File: cloudbuild-pr.yaml [hunk 1]
```diff
@@ -28,7 +28,6 @@ steps:
     - 'ESRA_URL=$_ESRA_URL'
     - 'KEYCLOAK_CONFIG_PATH=$_KEYCLOAK_CONFIG_PATH'
     - 'NAME_REQUEST_URL=$_NAME_REQUEST_URL'
-    - 'ONESTOP_URL=$_ONESTOP_URL'
```
Comment (severinbeauvais) [inline cloudbuild-pr.yaml:31]: I don't know if/where this in 1Password, nor whether any other project uses this key, but THIS project will no longer need it once this change has gone to Prod.

^^ Patrick

File: components/BCRegistriesAccount.vue [hunk 2]
```diff
@@ -211,7 +215,7 @@
       </v-card>
 
        <!-- BC Assessment -->
-      <v-card elevation="2" v-if="showBCAFlag">
+      <v-card v-if="showBCAFlag" elevation="2">
```
Comment (severinbeauvais) [inline components/BCRegistriesAccount.vue:218]: Lint fix (v-if must come first).

**Comment by severinbeauvais** [comment]
> /gcbrun

I think I don't have permission to run this. Doug, please?

**Review by doug-lovett — APPROVED** [review]

**Review by thorwolpert — COMMENTED** [review]
nice!

**Pull request merged**